data/src/main/java/com/android254/data/network/Constants.kt

```diff
@@ -17,5 +17,5 @@ package com.android254.data.network
 
 object Constants {
     const val BASE_URL = "https://api.droidcon.test/v1"
-    const val EVENT_SLUG = ""
+    const val EVENT_PATH_SEGMENT = "events/droidconke-2022-797"
```

Comment:
`EVENT_PATH_SEGMENT` -> `EVENTS_BASE_URL `

Reply:
Just for clarity, is this what you mean @chepsi?
 ```kotlin
const val EVENT_BASE_URL = "https://api.droidcon.test/v1/events/droidconke-2022-797"
```